src/styles/themes/light-theme.js

```diff
@@ -145,8 +145,7 @@ var LightTheme = {
         titleFontSize: 20,
         iconColor: 'rgba(0, 0, 0, .40)',
         separatorColor: 'rgba(0, 0, 0, .175)',
-        menuHoverColor: 'rgba(0, 0, 0, .10)',
-        menuHoverColor: Colors.white
```

Comment:
This was probably meant to make the color white in browsers not supporting rgba.
First, I'm not sure material-ui will work in IE8, the only browser that doesn't support rgba.
If we still need that, it's probably worth taking a look at how it's implemented in JSS: https://github.com/jsstyles/jss#multiple-declarations-with-identical-property-names